code/index.html

```diff
@@ -1,22 +1,180 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```

Comment:
For next project, perhaps give the page title a fun name. This is the text that's visible in the tab when on desktop. For this project Great News Delivery would be a great name 🥳